test/functional/tool_wallet.py

```diff
@@ -213,7 +213,7 @@ def test_salvage(self):
         self.assert_tool_output('', '-wallet=salvage', 'salvage')
 
     def run_test(self):
-        self.wallet_path = os.path.join(self.nodes[0].datadir, self.chain, 'wallets', 'wallet.dat')
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/20034#discussion_r496914396

> Should the `-wallet=wallet.dat` throughout this test use `self.wallet_data_filename` too then?

It effectively is. I think you are maybe only looking at the first commit, not the overall diff? The wallet.dat references are removed in the second commit, since more robust way to refer to wallets is by directory name not data file name. (The data file name used inside wallets is an implementation detail and referring to it is only supported in limited cases for backwards compatibility. Dropping references to the data file name lets the test work with all default_wallet_name values.)